Makefile

```diff
@@ -0,0 +1,57 @@
+.PHONY: $(MAKECMDGOALS)
+
+clean:
+	@rm -rf tests/tmp
+
+cleantests:
+	@rm -rf tests/tmp/.git
+	@rm -rf tests/tmp/empty
+	@rm -rf tests/tmp/CHANGELOG.md
+
+docs:
+	@mkdocs serve
+
+docs-deploy:
+	@mkdocs gh-deploy
+
+format:
+	@ruff format
+
+gen generate:
+	@bash -c 'source tests/helpers.sh && generate ${PWD} tests/tmp'
+
+reset-history: gen
+	@./reset-history.sh
+
+test: cleantests
+	@./runtests.sh
+
+changelog:
+	@git-changelog -Tio CHANGELOG.md -Bauto -c angular
+
+release:
+	@test -n "$(version)" || { echo "error: usage: make release version=x.y.z" >&2; exit 1; }
+	@git add CHANGELOG.md
+	@git commit -m "docs: Update changelog for version $(version)"
+	@git tag $(version)
+	@git push
+	@git push --tags
```

Comment:
Just in case: this is the Makefile for the template itself. I don't feel like we need a dry-run mode here. Feel free to elaborate :slightly_smiling_face:

Reply:
Ah, yes, thanks for the clarification. Nevermind.